### ISSUE TITLE
Bug Login dont accep Upper and Lowercase server name only lowercase

### ISSUE THREAD

**Issue by Iam2fast4u2catch** [nextcloud/android#2815]
### Actual behaviour
- Tell us what happens
Hi there can you squeeze out that nasty bug:

When you enter the server adress (i have my own Nextcloud Server) mixed in Upper AND Lowercase it says Server NOT found.

When you enter the server adress ALL in lowercase theres NO problem.

The owncloud app DONT have that problem if you enter the Serveradress mixed in upper AND lowercase it turns all to lowercase automatically and connect..

XhaHgaH.VuU dont work in your app neither in beta.

xhahgah.vuu works (for example) thx. Greets from Switzerland.
### Expected behaviour
- Tell us what should happen
 Auto turn upper AND lowercase servername ALL letters to lowercase like owncloud app do!
### Steps to reproduce
1. Enter Servername upper+Lowercase
2. For example HDddauHd.Com dont work
3. For example hdddauhd.com WORK..

### Environment data
Android version:
8.0
Device model: S7-Edge with S9+ CS-Rom

Stock or customized system:
Customized
Nextcloud app version:
Actual Beta AND normal
Nextcloud server version:
Actual aviable

#### Web server error log
```
Insert your webserver log here
```

#### Nextcloud log (data/nextcloud.log)
```
Insert your Nextcloud log here
```
**NOTE:** Be super sure to remove sensitive data like passwords, note that everybody can look here! You can use the Issue Template application to prefill some of the required information: https://apps.nextcloud.com/apps/issuetemplate

**Comment by tobiasKaminsky** [comment]
If this is the same as discussed in IRC, we traced it down to a problem in the local network, as it worked with my server and when using the mobile device via mobile data.
(If this issue is independent, please feel free to re-open).

**Comment by AndyScherzinger** [comment]
Also tried it with my server and uppercase/lowercase mixed works just fine.

**Comment by Iam2fast4u2catch** [comment]
Seems a split dns problem i tried to acess another cloud with the app with mixed lower and uppercase letters as server name it works for me. Seems a problem wheb i tried to acess the cloud on my internaly network. All lowercase works. So not really a bug. One last thing is i missed the tab clear sucessfully uploads list (in the android app) ,the list grow and grow...  and if a file allready exists only via webbbrowser it asks for replace/overwrite not in the app. kind regards M.L
Am 11.07.18, 13:05, Andy Scherzinger <notifications@github.com> schrieb:
Also tried it with my server and uppercase/lowercase mixed works just fine.

—You are receiving this because you authored the thread.Reply to this email directly, view it on GitHub, or mute the thread.
{"api_version":"1.0","publisher":{"api_key":"05dde50f1d1a384dd78767c55493e4bb","name":"GitHub"},"entity":{"external_key":"github/nextcloud/android","title":"nextcloud/android","subtitle":"GitHub repository","main_image_url":"https://assets-cdn.github.com/images/email/message_cards/header.png","avatar_image_url":"https://assets-cdn.github.com/images/email/message_cards/avatar.png","action":{"name":"Open in GitHub","url":"https://github.com/nextcloud/android"}},"updates":{"snippets":[{"icon":"PERSON","message":"@AndyScherzinger in #2815: Also tried it with my server and uppercase/lowercase mixed works just fine."}],"action":{"name":"View Issue","url":"https://github.com/nextcloud/android/issues/2815#issuecomment-404131303"}}}
[
{
"@context": "http://schema.org",
"@type": "EmailMessage",
"potentialAction": {
"@type": "ViewAction",
"target": "https://github.com/nextcloud/android/issues/2815#issuecomment-404131303",
"url": "https://github.com/nextcloud/android/issues/2815#issuecomment-404131303",
"name": "View Issue"
},
"description": "View this Issue on GitHub",
"publisher": {
"@type": "Organization",
"name": "GitHub",
"url": "https://github.com"
}
},
{
"@type": "MessageCard",
"@context": "http://schema.org/extensions",
"hideOriginalBody": "false",
"originator": "AF6C5A86-E920-430C-9C59-A73278B5EFEB",
"title": "Re: [nextcloud/android] Bug Login dont accep Upper and Lowercase server name only lowercase (#2815)",
"sections": [
{
"text": "",
"activityTitle": "**Andy Scherzinger**",
"activityImage": "https://assets-cdn.github.com/images/email/message_cards/avatar.png",
"activitySubtitle": "@AndyScherzinger",
"facts": [

]
}
],
"potentialAction": [
{
"name": "Add a comment",
"@type": "ActionCard",
"inputs": [
{
"isMultiLine": true,
"@type": "TextInput",
"id": "IssueComment",
"isRequired": false
}
],
"actions": [
{
"name": "Comment",
"@type": "HttpPOST",
"target": "https://api.github.com",
"body": "{\n\"commandName\": \"IssueComment\",\n\"repositoryFullName\": \"nextcloud/android\",\n\"issueId\": 2815,\n\"IssueComment\": \"{{IssueComment.value}}\"\n}"
}
]
},
{
"targets": [
{
"os": "default",
"uri": "https://github.com/nextcloud/android/issues/2815#issuecomment-404131303"
}
],
"@type": "OpenUri",
"name": "View on GitHub"
},
{
"name": "Unsubscribe",
"@type": "HttpPOST",
"target": "https://api.github.com",
"body": "{\n\"commandName\": \"MuteNotification\",\n\"threadId\": 355020643\n}"
}
],
"themeColor": "26292E"
}
]

**Comment by Iam2fast4u2catch** [comment]
Seems a split dns problem i tried to acess another cloud with the app with mixed lower and uppercase letters as server name it works for me. Seems a problem wheb i tried to acess the cloud on my internaly network. All lowercase works. So not really a bug. One last thing is i missed the tab clear sucessfully uploads list (in the android app) ,the list grow and grow...  and if a file allready exists only via webbbrowser it asks for replace/overwrite not in the app. kind regards M.L
Am 11.07.18, 11:48, Tobias Kaminsky <notifications@github.com> schrieb:
Closed #2815.

—You are receiving this because you authored the thread.Reply to this email directly, view it on GitHub, or mute the thread.
{"api_version":"1.0","publisher":{"api_key":"05dde50f1d1a384dd78767c55493e4bb","name":"GitHub"},"entity":{"external_key":"github/nextcloud/android","title":"nextcloud/android","subtitle":"GitHub repository","main_image_url":"https://assets-cdn.github.com/images/email/message_cards/header.png","avatar_image_url":"https://assets-cdn.github.com/images/email/message_cards/avatar.png","action":{"name":"Open in GitHub","url":"https://github.com/nextcloud/android"}},"updates":{"snippets":[{"icon":"DESCRIPTION","message":"Closed #2815."}],"action":{"name":"View Issue","url":"https://github.com/nextcloud/android/issues/2815#event-1727499378"}}}
[
{
"@context": "http://schema.org",
"@type": "EmailMessage",
"potentialAction": {
"@type": "ViewAction",
"target": "https://github.com/nextcloud/android/issues/2815#event-1727499378",
"url": "https://github.com/nextcloud/android/issues/2815#event-1727499378",
"name": "View Issue"
},
"description": "View this Issue on GitHub",
"publisher": {
"@type": "Organization",
"name": "GitHub",
"url": "https://github.com"
}
},
{
"@type": "MessageCard",
"@context": "http://schema.org/extensions",
"hideOriginalBody": "false",
"originator": "AF6C5A86-E920-430C-9C59-A73278B5EFEB",
"title": "Re: [nextcloud/android] Bug Login dont accep Upper and Lowercase server name only lowercase (#2815)",
"sections": [
{
"text": "",
"activityTitle": "**Tobias Kaminsky**",
"activityImage": "https://assets-cdn.github.com/images/email/message_cards/avatar.png",
"activitySubtitle": "@tobiasKaminsky",
"facts": [

]
}
],
"potentialAction": [
{
"targets": [
{
"os": "default",
"uri": "https://github.com/nextcloud/android/issues/2815#event-1727499378"
}
],
"@type": "OpenUri",
"name": "View on GitHub"
},
{
"name": "Unsubscribe",
"@type": "HttpPOST",
"target": "https://api.github.com",
"body": "{\n\"commandName\": \"MuteNotification\",\n\"threadId\": 355020643\n}"
}
],
"themeColor": "26292E"
}
]

**Comment by Iam2fast4u2catch** [comment]
Seems a split dns problem i tried to acess another cloud with the app with mixed lower and uppercase letters as server name it works for me. Seems a problem wheb i tried to acess the cloud on my internaly network. All lowercase works. So not really a bug. One last thing is i missed the tab clear sucessfully uploads list (in the android app) ,the list grow and grow...  and if a file allready exists only via webbbrowser it asks for replace/overwrite not in the app. kind regards M.L
Am 11.07.18, 11:48, Tobias Kaminsky <notifications@github.com> schrieb:
If this is the same as discussed in IRC, we traced it down to a problem in the local network, as it worked with my server and when using the mobile device via mobile data.
(If this issue is independent, please feel free to re-open).

—You are receiving this because you authored the thread.Reply to this email directly, view it on GitHub, or mute the thread.
{"api_version":"1.0","publisher":{"api_key":"05dde50f1d1a384dd78767c55493e4bb","name":"GitHub"},"entity":{"external_key":"github/nextcloud/android","title":"nextcloud/android","subtitle":"GitHub repository","main_image_url":"https://assets-cdn.github.com/images/email/message_cards/header.png","avatar_image_url":"https://assets-cdn.github.com/images/email/message_cards/avatar.png","action":{"name":"Open in GitHub","url":"https://github.com/nextcloud/android"}},"updates":{"snippets":[{"icon":"PERSON","message":"@tobiasKaminsky in #2815: If this is the same as discussed in IRC, we traced it down to a problem in the local network, as it worked with my server and when using the mobile device via mobile data.\r\n(If this issue is independent, please feel free to re-open)."}],"action":{"name":"View Issue","url":"https://github.com/nextcloud/android/issues/2815#issuecomment-404111237"}}}
[
{
"@context": "http://schema.org",
"@type": "EmailMessage",
"potentialAction": {
"@type": "ViewAction",
"target": "https://github.com/nextcloud/android/issues/2815#issuecomment-404111237",
"url": "https://github.com/nextcloud/android/issues/2815#issuecomment-404111237",
"name": "View Issue"
},
"description": "View this Issue on GitHub",
"publisher": {
"@type": "Organization",
"name": "GitHub",
"url": "https://github.com"
}
},
{
"@type": "MessageCard",
"@context": "http://schema.org/extensions",
"hideOriginalBody": "false",
"originator": "AF6C5A86-E920-430C-9C59-A73278B5EFEB",
"title": "Re: [nextcloud/android] Bug Login dont accep Upper and Lowercase server name only lowercase (#2815)",
"sections": [
{
"text": "",
"activityTitle": "**Tobias Kaminsky**",
"activityImage": "https://assets-cdn.github.com/images/email/message_cards/avatar.png",
"activitySubtitle": "@tobiasKaminsky",
"facts": [

]
}
],
"potentialAction": [
{
"name": "Add a comment",
"@type": "ActionCard",
"inputs": [
{
"isMultiLine": true,
"@type": "TextInput",
"id": "IssueComment",
"isRequired": false
}
],
"actions": [
{
"name": "Comment",
"@type": "HttpPOST",
"target": "https://api.github.com",
"body": "{\n\"commandName\": \"IssueComment\",\n\"repositoryFullName\": \"nextcloud/android\",\n\"issueId\": 2815,\n\"IssueComment\": \"{{IssueComment.value}}\"\n}"
}
]
},
{
"targets": [
{
"os": "default",
"uri": "https://github.com/nextcloud/android/issues/2815#issuecomment-404111237"
}
],
"@type": "OpenUri",
"name": "View on GitHub"
},
{
"name": "Unsubscribe",
"@type": "HttpPOST",
"target": "https://api.github.com",
"body": "{\n\"commandName\": \"MuteNotification\",\n\"threadId\": 355020643\n}"
}
],
"themeColor": "26292E"
}
]
